pkg/apis/servicecatalog/validation/instance_test.go

```diff
@@ -27,8 +27,8 @@ import (
 	"github.com/kubernetes-incubator/service-catalog/pkg/apis/servicecatalog"
 )
 
-const externalClusterServiceClassName = "test-serviceclass"
-const externalClusterServicePlanName = "test-plan"
+const ClusterServiceClassName = "test-serviceclass"
```

Comment:
should be kept private probably (camel-case)

Reply:
fixed